api/CanMsg.h

```diff
@@ -82,11 +92,45 @@ class CanMsg : public Printable
     return n;
   }
 
+
+  uint32_t getStandardId() const {
```

Comment:
Would it make sense to make the ID an ADT? That way you could have something like:

```
class CanId : public Printable
{
    virtual uint32_t toInt() const = 0;
};

class ExtendedCanId : public CanId
...

class StandardCanId: public CanId
...
```

This would allow libraries to be written that extend this to things like:

```
class DroneCANId : public ExtendedCanId
...

class CyphalId : public ExtendedCandId
...

class ARINC825 : public ExtendedCanId
...

// etc.
```